docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -32,7 +32,11 @@ For more information about KPA and HPA, see the documentation on [Supported Auto
         metadata:
           annotations:
             autoscaling.knative.dev/metric: "concurrency"
+            autoscaling.knative.dev/target-utilization-percentage: "70" 
     ```
+    !!! note <br/>
+             The `autoscaling.knative.dev/target-utilization-percentage` annotation for "Concurrency" 
+              <br/> specifies a percentage value
```

Comment:
These still aren't rendering properly...

<img width="437" alt="image" src="https://github.com/knative/docs/assets/44698588/1afc5339-5bf6-4379-b682-1d72734d79c3">

Indents blocks are four spaces, and you don't need the the `<br/>` tags at all.


```suggestion
    ```
    !!! note
        The `autoscaling.knative.dev/target-utilization-percentage` annotation for "Concurrency"  specifies a percentage value
```

It should look like this:

<img width="415" alt="image" src="https://github.com/knative/docs/assets/44698588/97d02173-606c-4d53-ae83-78f8f1356c97">

Reply:
Sure @psschwei Performed the suggested change. Thanks.